src/controller/kafka.rs

```diff
@@ -26,20 +26,31 @@ impl ConsumerContext for CustomContext {
 type LoggingConsumer = StreamConsumer<CustomContext>;
 
 pub fn create_consumer(brokers: &str, group_id: &str, topics: &[&str]) -> StreamConsumer<CustomContext> {
-    let context = CustomContext;
+	let context: CustomContext = CustomContext;
 
     let consumer: LoggingConsumer =  ClientConfig::new()
             .set("group.id", group_id)
             .set("bootstrap.servers", brokers)
             .set("enable.partition.eof", "false")
             .set("session.timeout.ms", "6000")
             .set("enable.auto.commit", "true")
+		.set("message.max.bytes", "1000000000")
+		.set("message.copy.max.bytes", "1000000000")
+		.set("receive.message.max.bytes", "2147483647")
+		.set("socket.send.buffer.bytes", "100000000")
+		.set("socket.receive.buffer.bytes", "100000000")
+		.set("queued.max.messages.kbytes", "2097151")
+		.set("fetch.message.max.bytes", "1000000000")
+		.set("max.partition.fetch.bytes", "1000000000")
+		.set("fetch.max.bytes", "2147483135")
         .set_log_level(RDKafkaLogLevel::Debug)
         .create_with_context(context)
-        .expect("Consumer creation failed");
+		.expect("Consumer creation failed")
```

Comment:
Please change this `expect` by a match instead and validate error. The `expect` make a exception if fail